src/utils/fetchClient.ts

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
It will be better if you remove all comments, especially this comment that disable ESlinter.